components/anthropic/actions/common/constants.mjs

```diff
@@ -1,6 +1,10 @@
 export default {
-  COMPLETION_MODELS: [
-    "claude-2",
-    "claude-instant-1",
+  MESSAGE_MODELS: [
+    "claude-3-opus-20240229",
+    "claude-3-sonnet-20240229",
+    "claude-3-haiku-20240307",
+    "claud-2.1",
```

Comment:
oops, I'm assuming this should be `claude-2.1`?

Reply:
https://github.com/PipedreamHQ/pipedream/pull/11164